clients/include/testing_gemv.hpp

```diff
@@ -43,15 +43,28 @@ hipblasStatus_t testing_gemv(const Arguments& argus)
         dim_y = N;
     }
 
+    hipblasLocalHandle handle(argus);
+
     // argument sanity check, quick return if input parameters are invalid before allocating invalid
     // memory
-    if(M < 0 || N < 0 || lda < 0 || incx <= 0 || incy <= 0)
+    bool invalid_size = M < 0 || N < 0 || lda < M || lda < 1 || !incx || !incy;
+    if(invalid_size || !M || !N)
     {
-        return HIPBLAS_STATUS_INVALID_VALUE;
+        // cuBLAS doesn't seem to work with nullptrs in some cases here
+        /*
+        hipblasStatus_t actual = hipblasGemvFn(
+            handle, transA, M, N, nullptr, nullptr, lda, nullptr, incx, nullptr, nullptr, incy);
+        EXPECT_HIPBLAS_STATUS(
+            actual, (invalid_size ? HIPBLAS_STATUS_INVALID_VALUE : HIPBLAS_STATUS_SUCCESS));
+        return actual;
+	*/
+        return invalid_size ? HIPBLAS_STATUS_INVALID_VALUE : HIPBLAS_STATUS_SUCCESS;
```

Comment:
Does this pass with rocBLAS? I guess it can be tested in rocBLAS then the test can be left out of hipBLAS.

Reply:
Yes, rocBLAS acts as I had expected, the commented code passes.

Reply:
Can you replace "cuBLAS dosen't seem to work" with "Some platforms do not work".

Reply:
> Can you replace "cuBLAS dosen't seem to work" with "Some platforms do not work".

Might be clearer to say "Only rocblas conforms to expected behaviour so commenting out."

Reply:
Yes, Torre's wording is better, more descriptive and will be easier to understand.